proposals/per_policy_webhook_configurations.md

```diff
@@ -0,0 +1,235 @@
+# Meta
+[meta]: #meta
+- Name: Policy-based Webhook Configurations
+- Start Date: 2023-12-12
+- Update data (optional):
+- Author(s): @realshuting
+
+# Table of Contents
+[table-of-contents]: #table-of-contents
+- [Meta](#meta)
+- [Table of Contents](#table-of-contents)
+- [Overview](#overview)
+- [Definitions](#definitions)
+- [Motivation](#motivation)
+- [Proposal](#proposal)
+- [Implementation](#implementation)
+- [Migration (OPTIONAL)](#migration-optional)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+- [Prior Art](#prior-art)
+- [Unresolved Questions](#unresolved-questions)
+- [CRD Changes (OPTIONAL)](#crd-changes-optional)
+
+# Overview
+[overview]: #overview
+
+Configure Kyverno resource webhook configurations based on policy settings.
```

Comment:
This overview isn't very descriptive and should be elaborated upon. Kyverno today does a form of this and so it needs to be explained.

Reply:
Will update it.

Reply:
>but we discussed having per-rule webhooks in the past which, although I don't see it stated, seems to be a complimentary decision. Is that not still on the table?

I linked to the other policy-based webhook configuration PR https://github.com/kyverno/kyverno/issues/8063 in the description. Is there any other prior issue logged for this?